tests/functional/codegen/test_abi_encode.py

```diff
@@ -31,7 +31,7 @@ def abi_encode(
     pet_metadata: bytes32,
     ensure_tuple: bool,
     include_method_id: bool
-) -> Bytes[260]:
+) -> Bytes[548]:
```

Comment:
What was the change here?

Reply:
The size bound calculation was broken for nested dynamic types. 53f64919820ad008c7ddb0d552ce28a6161d94f9 which fixes https://github.com/vyperlang/vyper/issues/2458